tests/protocol/test_music_db.py

```diff
@@ -732,15 +744,39 @@ def test_list(self):
         self.assertInResponse("OK")
 
     def test_list_foo_returns_ack(self):
-        self.send_request('list "foo"')
-        self.assertEqualResponse("ACK [2@0] {list} Unknown tag type: foo")
+        self.send_request('list "foO"')
+        self.assertEqualResponse("ACK [2@0] {list} Unknown tag type: foO")
 
     def test_list_without_type_returns_ack(self):
         self.send_request("list")
         self.assertEqualResponse(
             'ACK [2@0] {list} too few arguments for "list"'
         )
 
+    def test_list_all_supported_fields(self):
+        # Be nice to use pytest's parametrize here.
+        fields = [
+            "album",
+            "albumartist",
+            "artist",
+            "comment",
+            "composer",
+            "date",
+            "disc",
+            "file",
+            "filename",
+            "genre",
+            "musicbrainz_albumid",
+            "musicbrainz_artistid",
+            "musicbrainz_trackid",
+            "performer",
+            "title",
+            "track",
+        ]
+        for field in fields:
+            self.send_request(f'list "{field}"')
+            self.assertInResponse("OK")
```

Comment:
I think this should work:

```suggestion
    @pytest.mark.parametrize('field', [
        "album",
        "albumartist",
        "artist",
        "comment",
        "composer",
        "date",
        "disc",
        "file",
        "filename",
        "genre",
        "musicbrainz_albumid",
        "musicbrainz_artistid",
        "musicbrainz_trackid",
        "performer",
        "title",
        "track",
    ])
    def test_list_all_supported_fields(self, field):
        self.send_request(f'list "{field}"')
        self.assertInResponse("OK")
```

Reply:
Sadly, no, [you can't use parametrize with unittest.TestCase subclasses](https://docs.pytest.org/en/latest/unittest.html#pytest-features-in-unittest-testcase-subclasses).